org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -1288,7 +1322,7 @@ private void printRunningTotal (PrintData pd, int levelNo, int rowNo)
 				else if (m_group.isFunctionColumn(pdc.getAD_PrintFormatItem_ID(), PrintDataFunction.F_SUM))
 				{
 					pd.addNode(new PrintDataElement(pdc.getAD_PrintFormatItem_ID(), pdc.getColumnName(),
-						m_group.getValue(PrintDataGroup.TOTAL, pdc.getAD_PrintFormatItem_ID(), PrintDataFunction.F_SUM),
+						m_group.getValue(PrintDataGroup.TOTAL, pdc.getAD_PrintFormatItem_ID(), PrintDataFunction.F_SUM).toString(),
```

Comment:
why the need to add toString() here ?

Reply:
hello, I've added toString() there, because the third parameter of the PrintDataElement constructor is Serializable and m_group.getValue(...) returns Object

Reply:
should change getValue(...) to returnSerializable instead (same goes for addValue and getFunctionValue). toString() doesn't sounds right here.

Reply:
I see, thank you